ethereum/values/_common/lighthouse.yaml.gotmpl

```
@@ -0,0 +1,132 @@
+lighthouse:
+  {{ $scalingIndex := splitList "-" .Release.Name | last }}
+  executionClientUrl: http://erigon-{{ $scalingIndex }}-stateful-node:8551
+  jwt: {}
+    # Generate me with: openssl rand -hex 32
+    # Must match EL client jwt
+    # fromLiteral: "XXX_CHANGEME_XXX"
+    # Instead specifying the JWT in plain text using fromLiteral, you can also reference an existing Kubernetes Secret
+    # existingSecret:
+    #   name: eth-goerli-jwt
```

let's keep the same format as we do for nimbus for consistency:
```suggestion
    jwt:
      # Generate me with: openssl rand -hex 32
      # Must match EL client jwt
      fromLiteral: "XXX_CHANGEME_XXX"
      # Instead specifying the JWT in plain text using fromLiteral, you can also reference an existing Kubernetes Secret
      # existingSecret:
      #   name: eth-goerli-jwt
      #   key: jwt
```